r2/r2/templates/printablebuttons.html

```diff
@@ -451,17 +451,21 @@
     %if thing.can_block:
       ${self.banbuttons()}
         %if (not thing.was_comment or thing.thing.is_mention) and thing.thing.author_id != c.user._id and thing.thing.author_id not in c.user.enemies:
-          <li>
-            %if getattr(thing.thing, "from_sr", False):
-              %if getattr(thing.thing, "sr_blocked", False):
-                ${ynbutton(_("unblock subreddit"), _("unblocked"), "unblock_subreddit")}
-              %else:
-                ${ynbutton(_("block subreddit"), _("blocked"), "block", "hide_thing")}
-              %endif
-            %else:
-              ${ynbutton(_("block user"), _("blocked"), "block", "hide_thing")}
+          %if getattr(thing.thing, "from_sr", False):
+            %if not (thing.thing.subreddit.is_moderator(c.user) or c.user_is_admin):
```

Comment:
You can use `thing.user_is_moderator` - https://github.com/reddit/reddit/blob/master/r2/r2/models/link.py#L2031